xml/article_nfs_storage.xml

```diff
@@ -205,9 +205,16 @@
         following contents:
       </para>
 <screen>resource nfs {
-   device /dev/drbd0; <co xml:id="co-ha-quick-nfs-drbd-device"/>
-   disk   /dev/nfs/work; <co xml:id="co-ha-quick-nfs-drbd-disk"/>
-   meta-disk internal; <co xml:id="co-ha-quick-nfs-drbd-metadisk"/>
+   volume 0 { <co xml:id="co-ha-quick-nfs-drbd-volume"/>
+      device           /dev/drbd1 minor 1; <co xml:id="co-ha-quick-nfs-drbd-device"/>
+      disk             /dev/nfs/state; <co xml:id="co-ha-quick-nfs-drbd-disk"/>
+      meta-disk        internal; <co xml:id="co-ha-quick-nfs-drbd-metadisk"/>
+   }
+   volume 1 {
+      device           /dev/drbd2 minor 2;
```

Comment:
Could these be `/dev/drbd0` and `/dev/drbd1`? I just wonder if having mismatched numbers for the volumes and devices might potentially trip anyone up. Could be easy to mistype the wrong number.

Reply:
Aha, exactly. Coincidentally, I was reviewing another bug and correlated to this about an hour ago.  

Yes, your suggestion is better. For example,

volume 0 <--> drbd0 <--> /nfs/state
volume 1 <--> drbd1 <--> /nfs/share

Reply:
Just FYI:  Besides section 4.1, section 4.2 mentions drbd1/2 as well.

Reply:
I found conflicting info about whether `minor x` can start at 0 or 1, but found consistent info that if you specify a name then you can leave out the minor. So I'm removing `minor 1` and `minor 2`, but do let me know if it actually needs to stay in.